README.md

```diff
@@ -30,3 +30,12 @@ enigo.key_sequence_parse("{+CTRL}a{-CTRL}{+SHIFT}Hello World{-SHIFT}");
 ```
 
 for more look at examples
+
+Runtime dependencies
+--------------------
+
+Linux users may have to install libxdo-dev:
+
+```Bash
+apt-get install libxdo-dev
```

Comment:
`apt` is the recommended binary for Ubuntu I think, and `apt-get` is mostly for compatibility. Would also be great to specify "For example, on Ubuntu...", so people on other systems know that exact command might not work for them

Reply:
Agreed, thanks!